### ISSUE TITLE
[feature request] Catalog scanner

### ISSUE THREAD

**Issue by polak0v** [Dimillian/ACHNBrowserUI#315]
Hi, and thanks to the team for this great job, so far it has been my favorite companion app for ACNH!

I was wondering if you plan to integrate https://github.com/EhsanKia/CatalogScanner (like ACNH.guide does it for example).
It would be useful to add our collection list to the app (with also a button to add all of them directly, instead of checking each).

**Comment by ericlewis** [comment]
I actually tried to do a version of this. Very cool idea and it will be considered.

**Comment by Dimillian** [comment]
Yeah! This is definitely something we've been thinking about and that @ericlewis started to build. Would love to have that built in!
